src/web/theme/pages/Home/Home.js

```diff
@@ -1,20 +1,24 @@
 import React from "react";
 import HomeQuery from "./HomeQuery.gql";
 import EnhanceHome from "./EnhanceHome";
-import ProductList from "theme/modules/ProductList";
-import HomeBanner from "theme/modules/HomeBanner";
-import LoadingArea from "theme/ui/molecules/LoadingArea";
 import Page from "theme/ui/templates/Page";
+import { H1 } from "theme/ui/atoms/Typography/Heading";
+import createMediaUrlFromPath from "../../../utils/createMediaUrlFromPath.js";
+import "./Home.scss";
 
-const Home = ({ store, category, loading }) => (
-  <Page>
-    <HomeBanner storeName={store && store.name} />
-    {loading ? (
-      <LoadingArea>Loading products…</LoadingArea>
-    ) : (
-      <ProductList products={category.layer.products} />
-    )}
-  </Page>
-);
+const Home = ({ category, loading }) =>
+  console.log(category) || (
+    <Page>
+      <div className="home">
+        <img
+          src={createMediaUrlFromPath(
+            "/media/catalog/category/adrien-tutin-379257-unsplash_1.png"
+          )}
+        />
+        <H1>History is</H1>
+        <H1>closer than you think!</H1>
```

Comment:
That is not great :laughing: the goal here is simply to have the text on two lines ? a simple <br /> is more correct

Reply:
For more details:
- It's sanctionned by SEO. You should have one and only one H1 per page.
- This can be problematic for screen readers.